### DISCUSSION TITLE
Is there a way I can add a heading for the sidebar but in bigger text ?

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#1141] [Q&A]
Currently the sidebar looks like this 

![image](https://github.com/user-attachments/assets/7bade5ad-6148-4057-9e23-8ff9510e5d4c)

And what I want to do is add an heading bigger than the one which can be written in the meta.json file. Is there a way I can do that?

**Comment by Gitstar-OC** [comment]
@fuma-nama  Also is there any way that I can pass down the `primary` color for each of the different folder that I have ? 

![image](https://github.com/user-attachments/assets/c1790708-2c0e-4c98-875e-576228f18800)

So that it will be changed based on the folder I am using, it will be good if there is a way or I will need to use the `usePathName` hook.

**Comment by fuma-nama** [comment]
Replace the components with `sidebar.components` prop

**Comment by Gitstar-OC** [comment]
I am not using the fumadocs 14. Is it possible to do so in previous versions without updating ?  

![image](https://github.com/user-attachments/assets/6480df55-df5a-48cb-ad89-940eed0492a7)
